src/main/java/at/aau/serg/websocketdemoserver/messaging/dtos/CardPlayRequest.java

```diff
@@ -8,9 +8,9 @@ public class CardPlayRequest {
     @JsonProperty("lobbyCode")
     private String lobbyCode;
     @JsonProperty("userID")
-    private String playerID;
+    private String userID;
     @JsonProperty("color")
     private String color;
     @JsonProperty("value")
-    private Integer value;
+    private String value;
```

Comment:
Warum auf string geändert? Für Sonderkarten?

Reply:
Value ist zwar in der Karte ein Integer, bei den derzeitigen DTOs werden, bis auf CardPlayRequest, immer Strings versendet, war nur zur Vereinheitlichung gedacht, kann aber wieder auf Integer geändert werden wenn es verwirrend ist. Für Sonderkarten müssen wir die DTOs wahrscheinlich noch um CardType erweitern. :)